OpenTESArena/src/Game/Game.cpp

```diff
@@ -36,7 +34,8 @@ Game::Game()
 	this->options = OptionsParser::parse();
 
 	// Initialize virtual file system using the Arena path in the options file.
-	VFS::Manager::get().initialize(std::string(this->options->getArenaPath()));
+    std::string arena = File::toString(this->options->getArenaPath());
```

Comment:
Maybe I want to add a `File::exists()` method that will check if the `ARENA` folder exists.